frontend/src/metabase/visualizations/visualizations/LinkViz/LinkViz.unit.spec.tsx

```diff
@@ -227,16 +227,57 @@ describe("LinkViz", () => {
       });
 
       const infoIcon = screen.getByLabelText("info icon");
-
       await userEvent.hover(infoIcon);
 
       await waitFor(() => {
-        expect(screen.getByText(longDescription)).toBeInTheDocument();
+        expect(screen.getByText("Bold text")).toHaveStyle({
+          "font-weight": "bold",
+        });
       });
 
-      expect(screen.getByText(longDescription)).toBe(
-        screen.getByTestId("wrapped-tooltip"),
```

Comment:
This test verified content was wrapped by having an element with this test id which is not very reliable anyway. Now the markdown renderer wraps text

Reply:
#54405

Reply:
Could be https://github.com/metabase/metabase/pull/53447 as well, looking at it